bin/magellan

```diff
@@ -140,7 +140,7 @@ if (settings.aggregateScreenshots) {
 //
 var tests = getTests(testFilters.detectFromCLI(margs.argv));
 
-if (!tests) {
+if (!tests || !tests.length) {
```

Comment:
I'd vote for [`_.isEmpty`](https://lodash.com/docs#isEmpty) here since we have lodash available